.ci/Jenkinsfile

```diff
@@ -47,10 +47,44 @@ pipeline {
         }
       }
     }
+    stage('Parallel'){
```

Comment:
I guess it would be more stages in the future to run in parallel, isn't it?

Reply:
Yes, I initially created all the stages there, but I reverted it and created this PR with only the Linux stage for the time being.